docs/autodoc_build/divvy.md

```diff
@@ -174,7 +166,7 @@ D.pop(k[,d]) -> v, remove specified key and return the corresponding value.
 
 If key is not found, d is returned if given, otherwise KeyError is raised.
 ```python
-def pop(self, key, default=<object object at 0x7f5d0764d030>):
+def pop(self, key, default=<object object at 0x7f2dffda3030>):
 ```
```

Comment:
what is this object notation? it's new to me

Reply:
No idea...this is inherited from `MutableMapping` I think